CHANGELOG.md

```diff
@@ -3,6 +3,52 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.22.2
+
+_Nov 7, 2024_
+
+We'd like to offer a big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```

Comment:
We are missing the highlights before merging,

Reply:
Yeah, not sure we have anything exciting to announce 😅 

Should we mention anything about the v8 alpha?

Reply:
> Should we mention anything about the v8 alpha?

I think we need to make the changelog of `v7.22.2` part of `v8.0.0-alpha.0` diff, so announcing it earlier might not make sense as it'll be released afterward. Wdyt?

Reply:
It would be nice to add a note about the alpha yes

Other than that, the usual `- 🐞 Bugfixes` I guess :laughing:

Reply:
> I think we need to make the changelog of v7.22.2 part of v8.0.0-alpha.0 diff, so announcing it earlier might not make sense as it'll be released afterward. Wdyt?

The changelog also gets added to the Github release, which is displayed when  you have Renovate or Dependabot.
So at least for the GIthub release I think it should be on v7.22.2.

For the CHANGELOG.md file, we might just need a not saying that v7 is entering maintenance mode.